docs/api-reference/mapbox/overview.md

```diff
@@ -1,6 +1,8 @@
 # @deck.gl/mapbox
 
-This module makes it easy to use deck.gl as native layers and controls in the Mapbox GL JS ecosystem. 
+This module makes it easy to use deck.gl as native layers and controls in the Mapbox and Maplibre GL JS ecosystem. 
+
+> For brevity, the term 'mapbox-gl' will refer collectively to any library that is compatible with Mapbox GL JS, including but not limited to maplibre-gl. Known exceptions will be clearly marked. For more, see [Compatibility with Mapbox GL JS forks](../../developer-guide/base-maps/using-with-mapbox.md#compatibility-with-mapbox-gl-js-forks)
```

Comment:
I would probably keep typing out `mapbox-API compatible library` in the text rather than focus on brevity.